.travis.yml

```diff
@@ -14,6 +14,9 @@ stages:
   - test
   - name: deploy
     if: type != pull_request
+  - name: push-chart
+    # require the tag name to match a regular expression
+    if: tag =~ ^v
```

Comment:
should tag be `$TRAVIS_TAG` ?

Reply:
The conditional stages/jobs have their own syntax

https://docs.travis-ci.com/user/conditional-builds-stages-jobs/

Reply:
cool. didnt know